tests/columns/test_datecolumn.py

```diff
@@ -77,8 +77,9 @@ class Table(tables.Table):
             date = tables.DateColumn()
             date_linkify = tables.DateColumn(accessor="date", linkify=isoformat_link)
 
-        table = Table([{"date": date(2012, 9, 12)}])
-        self.assertEqual(table.rows[0].get_cell_value("date"), "09/12/2012")
+        dt = date(2012, 9, 12)
+        table = Table([{"date": dt}])
+        self.assertEqual(table.rows[0].get_cell_value("date"), dt.isoformat())
```

Comment:
In unit tests we should use literal expected values when possible:

```suggestion
        table = Table([{"date": date(2012, 9, 12)}])
        self.assertEqual(table.rows[0].get_cell_value("date"), "2012-09-12")
```

Please also adjust the other tests.